doc/storagedriver/azure.md

```diff
@@ -10,6 +10,7 @@ The following parameters must be used to authenticate and configure the storage
 * `accountname`: Name of the Azure Storage Account.
 * `accountkey`: Primary or Secondary Key for the Storage Account.
 * `container`: Name of the root storage container in which all registry data will be stored. Must comply the storage container name [requirements][create-container-api].
+* `domain`: (optional) Domain name for the Storage Service. Defaults to `core.windows.net`. For example, `core.chinacloudapi.cn` would be the correct value for Azure in China and `core.usgovcloudapi.net` would be the correct value for Azure Government.
```

Comment:
@ahmetalpbalkan What does it say in the azure documentation? It should match that.

In Go land (and registry land), we'd almost always call this a "host", since it lacks a port. I see no reason to enforce that here if it's well-defined in azure documents as something else.